include/ftxui/screen/color.hpp

```diff
@@ -1,6 +1,7 @@
 #ifndef FTXUI_SCREEN_COLOR
 #define FTXUI_SCREEN_COLOR
 
+#include <cassert>
```

Comment:
Could you put the implementation and this header inside the .cpp file?

Reply:
done.